platform/src/main/java/platform/pageobjects/MyPage/Membership.java

```diff
@@ -0,0 +1,134 @@
+package platform.pageobjects.MyPage;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utils.CommonFunctions;
+import utils.XLHandler;
+
+public class Membership {
+
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(Membership.class.getName());
+	int waitTime = 5;
+	String[] shipmentdata;
+	String freeuser;
+
+	public Membership(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//a[@class='link js-not-link']")
+	public WebElement accountInformation;
+
+	@FindBy(xpath = "//div[@class='header__control']//li[2]//a[1]")
```

Comment:
Please remove absolute xpath and replace with relative wherever applicable

Reply:
@csonparote San
Do I need to change both the xpath ?

Reply:
I have changed the xpath.